src/cast_core/src/RollingCollections.cpp

```diff
@@ -50,25 +52,18 @@ struct RunOperation {
     RollingCollectionNames& rollingCollectionNames;
 };
 
+static long getMillisecondsSinceEpoch() {
+    return std::chrono::duration_cast<std::chrono::milliseconds>(
+               std::chrono::system_clock::now().time_since_epoch())
+        .count();
```

Comment:
this answers my question in the TODO I added - need to change this to `metrics::clock::now()`